script/download-node.js

```diff
@@ -25,7 +25,7 @@ const downloadFileToLocation = function(url, filename, callback) {
   stream.on('error', callback);
   const requestStream = request.get(url)
   requestStream.on('response', function(response) {
-    if (response.statusCode == 404) {
+    if (response.statusCode === 404) {
```

Comment:
"For fun" (out of curiosity), I checked that the `statusCode` is in fact a number, and not a string, when the URL is wrong. (I deliberately edited the hard-coded target URL to be wrong, logged the `typeof response.statusCode` here, and I got:

`typeof response.statusCode is: number`

Script behaves as expected with `=== 404` as opposed to `=== "404"` which bugs out.

Overly thorough way of saying this change is good 👍

Reply:
Thanks a ton for double checking! My intuition in this space said they should be numbers and I didn't question it. So I'm happy you've actually checked